fix: update pricing page table [sc-12264]

ndom91 opened the pull request:
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [ ] Docs
* [ ] Learn
* [x] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

<!-- You can erase any parts of this template not applicable to your Pull Request. -->
## Notes for the Reviewer
<!-- Anything the reviewer should pay extra attention to. -->

- Pricing changes

## New Dependency Submission
<!-- Please explain here why we need the new dependency. -->

ndom91 commented:
What do yall think about putting this calculator stuff into the dropdown thing as well? Since its now only pertinent to the "prepay" info in there any way.

For example:
![image](https://user-images.githubusercontent.com/7415984/199768193-5049c88d-0513-4fb9-9f0a-6d73ea2fbc7e.png)

![image](https://user-images.githubusercontent.com/7415984/199768141-7198dc9f-a597-49c2-bfff-54600a368855.png)

ndom91 commented:
I will set this back to "Draft". 

We can use this branch/PR as a starting point later on when we want to also deploy the "Team" and "Enterprise" changes.

ndom91 commented:
The first PR, changing only the [`Hobby` plan column](https://github.com/checkly/checklyhq.com/pull/575) has been merged.

This PR also removes the `Pay-as-you-go / prepaid` column in favor of the `Teams` plan. Again, DO NOT MERGE this PR until the actual app / api changes have been made (currently WIP).

ndom91 commented:
![image](https://user-images.githubusercontent.com/7415984/204002149-8d3c3584-ee36-4dc8-b416-b3cc9645213b.png)

ndom91 commented:
Current looks:

![image](https://user-images.githubusercontent.com/7415984/204529323-7085307c-1b01-4561-bbd3-04aeff2c1e58.png)

maxigimenez reviewed (changes requested):
@ndom91 This alignment seems off

![Screenshot 2022-11-29 at 13 53 11](https://user-images.githubusercontent.com/1178139/204547197-7b9e4a5c-1931-4b19-83b6-52d0f1ca8dd8.png)

ndom91 commented:
Good eye on those two. Fixed :+1:

maxigimenez commented:
@ndom91 wrong icon on private-locations team plan, then LGTM! 😄

ndom91 commented:
> @maxigimenez Good eye on those two. Fixed +1



> @ndom91 wrong icon on private-locations team plan, then LGTM! smile

Ahh yes, good eye! Fixed.
